apps/src/netsim/NetSimLogWidget.js

```diff
@@ -74,7 +74,7 @@ NetSimLogWidget.prototype.log = function (message) {
       scrollArea[0].scrollHeight - scrollArea[0].scrollTop <=
       scrollArea.outerHeight();
 
-  $('<div>').html(message).appendTo(this.scrollArea_);
+  this.scrollArea_.val(this.scrollArea_.val() + message + '\n');
```

Comment:
nit: might use scrollArea instead of this.scrollArea_ since you created a local for it.